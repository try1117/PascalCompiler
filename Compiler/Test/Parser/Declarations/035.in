program test;
begin
  write('a');
  write(integer('a'));
  write(double('a'));
  write(char('a'), 48, char(48));
end.
