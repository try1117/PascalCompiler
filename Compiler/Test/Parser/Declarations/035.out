test : function()
   resultType : Nil

test declarations:
|-- Statements
|            |-- Write
|            |       |-- 'a'
|            |-- Write
|            |       |-- 97
|            |-- Write
|            |       |-- 97.000000
|            --- Write
|                    |-- 'a'
|                    |-- 48
|                    --- '0'

